queries/dune_v2/period_totals.sql

```diff
@@ -3,7 +3,7 @@ with
 execution_costs as (
     SELECT
         success,
-        sum((gas_used * gas_price)/ pow(10, 18)) as gas_cost_eth
+        sum((gas_price * gas_used) / pow(10, 18)) as gas_cost_eth
```

Comment:
Does this change do anything?

Reply:
Yes. This change is dividing within the sum like you mentioned in your comment. 

The reason this is less likely to overflow is because we're dealing with much smaller numbers. Although still technically possible that a single term overflows.

Reply:
Oh. Sorry wait. This query already had division in the sum. I guess it doesn't do anything.